web_external/ImagesGallery/Listing/ImagesPagingPane.js

```diff
@@ -65,52 +71,55 @@ isic.views.ImagesPagingPane = Backbone.View.extend({
         var hasPaging = this.images.hasPreviousPage() || this.images.hasNextPage();
 
         // Disable / enable the appropriate paging buttons
-        this.$('#isic-images-seekPrev, #isic-images-seekFirst')
-            .toggleClass('disabled', !this.images.hasPreviousPage());
-
-        this.$('#isic-images-seekNext, #isic-images-seekLast')
-            .toggleClass('disabled', !this.images.hasNextPage());
+        this.$('#isic-images-paging-seek-prev, #isic-images-paging-seek-first')
+            .prop('disabled', !this.images.hasPreviousPage());
+        this.$('#isic-images-paging-seek-next, #isic-images-paging-seek-last')
+            .prop('disabled', !this.images.hasNextPage());
```

Comment:
Girder 2.0 adds the convenience function `girderEnable()` that we should investigate once the plugin is converted.